.github/workflows/pr-tests.yaml

```diff
@@ -6,7 +6,21 @@ on:
       - 'CHANGELOG.md'
       - 'CHANGELOG_PENDING.md'
 
-permissions: read-all
+permissions:
+  actions: read
+  attestations: read
+  checks: read
+  contents: read
+  deployments: read
+  id-token: write
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  pull-requests: read
+  repository-projects: read
+  security-events: read
+  statuses: read
```

Comment:
Do you need to specify all these? Or is this just because it was previously read-all and it's not worth trimming down?

Reply:
Yeah this is semantically equivalent to the prior `read-all` + `id-token: write`.